.gitignore

```diff
@@ -7,17 +7,31 @@
 /.foreman
 /coverage/*
 /db/*.sqlite3
+
 /log/*
 /tmp/*
 !/log/.keep
 !/tmp/.keep
+
 /spec/example_app/log/*
 /spec/example_app/tmp/*
 !/spec/example_app/log/.keep
 !/spec/example_app/tmp/.keep
 /spec/example_app/public/system
 /spec/example_app/public/assets
+
+/spec/example_app/app/assets/builds/*
+!/spec/example_app/app/assets/builds/.keep
```

Comment:
> This is the dir I get builds in. Am I missing something?

@pablobm Yes, this is the dir we get the builds in. I thought of not committing the compiled assets to Git because that doesn't seem necessary. Compiled assets will be in the gem when we build it.

No strong opinions. Let know if you'd like to commit them to Git